lib/ontology_saver.rb

```diff
@@ -104,16 +104,16 @@ def find_existing_ontology(ontology_version_options)
       without_parent.first
   end
 
-  def create_ontology(filepath, locid)
+  def create_ontology(filepath, basepath)
```

Comment:
argument `basepath` is redundant. it is set in `ontology.basepath = File.basepath(filepath)`